provisioning/transport/http/src/http.ts

```diff
@@ -226,6 +230,10 @@ export class Http extends EventEmitter implements X509ProvisioningTransport, Tpm
         if (response.statusCode < 300) {
           /* Codes_SRS_NODE_PROVISIONING_HTTP_18_039: [ If the Http request succeeds, `queryOperationStatus` shall call `callback`, passing a `null` error along with the `result` and `response` objects. ] */
           callback(null, result, response, this._config.pollingInterval);
+        } else if (response.statusCode >= 429) {
+          /*Codes_SRS_NODE_PROVISIONING_HTTP_06_007: [ If the `queryOperationStatus` response contains a status code >= 429, the result.status value will be set with `assigning` and the callback will be invoked with *no* error object. ] */
+          result.status = 'assigning';
+          callback(null, result, response, this._config.pollingInterval);
```

Comment:
>result [](start = 25, length = 6)

what is this in that case? i believe it's an error. the polling state machine is going to emit an operationStatus event with that error instead of a result. is this OK?

Reply:
looks like no one is using those events from the polling state machine.... maybe we should stop emitting them if they have no value.

---
In reply to: [234755384](https://github.com/Azure/azure-iot-sdk-node/pull/430#discussion_r234755384) [](ancestors = 234755384)